buildSrc/src/main/resources/checkstyle.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+    "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+    "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+
+<module name="Checker">
+    <property name="charset" value="UTF-8" />
+
+    <!-- Checks Java files and forbids empty Javadoc comments -->
+    <module name="RegexpMultiline">
+        <property name="format" value="\/\*[\s\*]*\*\/"/>
+        <property name="fileExtensions" value="java"/>
+        <property name="message" value="Empty javadoc comments are forbidden"/>
+    </module>
+
+    <module name="TreeWalker">
+        <!-- Its our official line length! See checkstyle_suppressions.xml for the files that don't pass this. For now we
+          suppress the check there but enforce it everywhere else. This prevents the list from getting longer even if it is
+          unfair. -->
+        <module name="LineLength">
+            <property name="max" value="100"/>
+        </module>
+
+        <module name="AvoidStarImport" />
+        <!-- Doesn't pass but we could make it pass pretty quick.
+        <module name="UnusedImports">
+          The next property is optional. If we remove it then imports that are
+          only referenced by Javadoc cause the check to fail.
+          <property name="processJavadoc" value="true" />
+        </module>
+        -->
+
+        <!-- Non-inner classes must be in files that match their names. -->
+        <module name="OuterTypeFilename" />
+        <!-- No line wraps inside of import and package statements. -->
+        <module name="NoLineWrap" />
+        <!-- Each java file has only one outer class -->
+        <module name="OneTopLevelClass" />
+        <!-- The suffix L is preferred, because the letter l (ell) is often
+        hard to distinguish from the digit 1 (one). -->
```

Comment:
The comment here plus the name of this check "UpperEll" was not enough to inform me about the behavior. Googling around in search of understanding, this seems to be about requiring an uppercase "L" specifically on long literals? Like "1000L" vs "1000l"

Can you add in the comment that this is for `long` literals?